tiledb/api/c_api/query_field/test/unit_capi_query_field.cc

```diff
@@ -76,25 +76,31 @@ void QueryFieldFx::write_sparse_array(const std::string& array_name) {
 
   throw_if_setup_failed(tiledb_query_set_layout(ctx, query, TILEDB_UNORDERED));
 
-  int32_t a[] = {1, 2, 3, 4, 5, 6, 7, 8, 9, 10};
-  int32_t b[] = {1, 2, 3, 4, 5, 6, 7, 8, 9, 10};
-  uint64_t a_size = 10 * sizeof(int32_t);
-  uint64_t b_size = 10 * sizeof(int32_t);
-
-  int64_t d1[] = {1, 2, 3, 4, 5, 6, 7, 8, 9, 10};
-  int64_t d2[] = {1, 1, 1, 1, 1, 1, 1, 1, 1, 1};
-  uint64_t d1_size = 10 * sizeof(int64_t);
-  uint64_t d2_size = 10 * sizeof(int64_t);
+  int32_t a[] = {1, 2, 3, 4, 5, 6, 7, 8, 9};
```

Comment:
The last row was unused so I elected to remove it to avoid future confusion.

(e.g. a developer expecting a sum of 55 rather than 45, you can probably guess who did that)